client/src/components/polls/PollModal.js

```diff
@@ -1,16 +1,34 @@
-import React from 'react';
+import React, { useCallback, useEffect, useContext } from 'react';
 import axios from 'axios';
 import { makeStyles } from '@material-ui/core/styles';
 import { Button, Modal, TextField } from '@material-ui/core';
+import { useDropzone } from 'react-dropzone';
+import { UserContext } from '../UserContext';
 
 export default function PollModal(props) {
   const classes = useStyles();
-  const [open, setOpen] = React.useState(false);
+  const [open, setOpen] = React.useState(true);
   const [title, setTitle] = React.useState('');
   const [image1, setImage1] = React.useState('');
   const [image2, setImage2] = React.useState('');
   const [loading, setLoading] = React.useState(false);
   const [userId, setUserId] = React.useState('1234');
+  const user = React.useContext(UserContext);
+
+  useEffect(() => {
+    setUserId(user._id);
```

Comment:
You actually don't need useEffect and setUserId on this component, because it won't change.
Just remove both and add:
`const userId = user._id`